h2d/domkit/BaseComponents.hx

```diff
@@ -4,7 +4,7 @@ import domkit.CssValue;
 
 typedef FlowBg = { tile : #if macro Bool #else h2d.Tile #end, borderL : Int, borderT : Int, borderR : Int, borderB : Int, ?color : Int }
 
-class CustomParser extends CssValue.ValueParser {
+class CustomParser extends domkit.CssValue.ValueParser {
```

Comment:
This one relied on very wonky partial resolution: We're in `h2d.domkit`, and Haxe 4 would somehow resolve `CssValue` to `domkit.CssValue`, which is just wrong.